lib/paginate_firestore.dart

```diff
@@ -252,6 +256,54 @@ class _PaginateFirestoreState extends State<PaginateFirestore> {
 
     return listView;
   }
+
+  Widget _buildPageView(PaginationLoaded loadedState) {
+    var pageView = CustomScrollView(
```

Comment:
Do we need header and footer for page view? If we do, can we use Column instead of CustomScrollView?

Reply:
I tried a bunch of different things with this. Column wouldn't work, and just returning a SliverToBoxAdapter/SliverFillRemaining, but apparently a multi child sliver thing is needed. See my solution in the newest commit. I moved all the customizations (scrollDir, physics, etc.) down to pageview level and removed them from CustomScroll.

Reply:
Are you sure about the column? Have you tried something like this...

```dart
Column(
  children:[
    Header(),
    Expanded(child: PreloadedPageView(),)
    Footer(),
  ]
)
```

Reply:
That does work, however, when adding a Header, it's static. It doesn't get scrolled away since it's just in a column wth no scrollview outside of it. I think the idea here of a header would be to have it still be at the beginning of the list, and then disappear. Any header that they want to be static could just be done outside the paginate_firestore widget.

Reply:
Makes sense.